billing/scout-billing.md

```diff
@@ -0,0 +1,44 @@
+---
+title: Docker Scout billing
+description: Learn how to buy Docker Scout Team in Docker Hub and manage your subscription
+keywords: payments, billing, subscription
+---
+
+{% include scout-early-access.md %}
+
+Docker Scout lets users secure their software supply chain and continuously observe and improve their security posture. Docker Scout is free for up to 3 repositories. You can buy Docker Scout Team to turn on Docker Scout for additional repositories. In this section, learn how to buy Docker Scout Team in Docker Hub for your personal account or for an organization.
+
+## Personal account
+
+1. Select your username in the top-right corner of Docker Hub.
+2. From the drop-down menu select **Billing**.
+3. In the **Plan** section, navigate to the **Docker Scout** plan section. The default plan is **Docker Scout Free**. 
+4. Next to the plan name, select **Buy now**.
+5. From the **Purchase Scout Team** drop-down menu, select the name of your personal account.
```

Comment:
**suggestion**: I think this could be made a little more clear; we have section headers here - it'd be nice to mention that the account is underneath "Your Account" and not "Organizations"
<img width="303" alt="image" src="https://github.com/docker/docs/assets/104848013/cf30d7bb-ccb5-41ee-90ff-1b400662eb71">